docs/root/configuration/other_features/graphite_statsd_stat_sink.rst

```diff
@@ -0,0 +1,7 @@
+.. _config_stat_sinks_graphite_statsd:
+
+Graphite+Statsd Stat Sink
+==============
```

Comment:
underline should match length of title in rst

Reply:
thank you!